src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/PaymentServiceConstants.java

```diff
@@ -0,0 +1,22 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
+
+/**
+ * 
+ * PaymentServiceConstants
+ * 
+ * @author 176899
```

Comment:
do we want this or our github handle or nothing?

Reply:
I can change this if you prefer. This comes from my NTTData username. Would you prefer shaunmillargov (github) or smillar, or ??